### PR TITLE
USHIFT-1242: Use t2.xlarge in AWS jobs

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-ref.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ ref:
     documentation: |-
       BYO AMI for EC2
   - name: EC2_INSTANCE_TYPE
-    default: t2.2xlarge
+    default: m6i.xlarge
```
Comment (dhellmann) [inline ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-ref.yaml:19]: I think the PR message is out of date with the actual change.

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced
/hold

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Pull request closed without merging**